backend/db/config.py

```diff
@@ -103,7 +109,7 @@ def get_pg_connect_url(local=False):
 # - Last updated: 2024/04/07
 # todo: are the 'module names' part of the DDL file names the same as the keys in this dictionary? If so, write that
 #  down here as a comment. Could be useful for dynamically selecting the DDL files.
-DERIVED_TABLE_DEPENDENCY_MAP = {
+DERIVED_TABLE_DEPENDENCY_MAP: Dict[str, List[str]] = {
```

Comment:
New relationships:
- concept_depth
- root_concepts

Vocab refresh wil now update these

Reply:
FYI, when this gets updated, in order for the vocab refresh to know about these changes, I do a `rebase` of the `shadow-develop` branch on: 
- https://github.com/shadow-bids/termhub/

The main reason I do a `rebase` instead of a merge is so that there is a single commit difference between these branches, and I like it to show up at the top of the history. That commit deactivates every GH action, except for vocab/counts, which it activates.